README.md

```diff
@@ -2738,6 +2750,10 @@ Enable or Disable Layer3 forwarding for packets with peer gateway-mac. Valid val
 ##### `peer_gateway_exclude_vlan`
 Interface vlans to exclude from peer gateway functionality. Valid value is a string of integer ranges from 1..4095. This parameter is available only in Nexus 5000, Nexus 6000 and Nexus 7000 series. There is no default value.
 
+##### `port_channel_limit`
+In vPC+ mode, enable or disable the port channel scale limit of
+244 vPCs.  Valid values are true, false or default
```

Comment:
No , this cannot change as it is limited by the protocol field size of 8 bits (sub-switch-id). This exists only in N7K (with F2 and lower rev modules). In other platforms, there is no option to use sub-switch-id and so there is no such limit. They use a second lookup on the mac-table to determine the port of exit.